src/librustdoc/fold.rs

```diff
@@ -56,6 +56,10 @@ crate trait DocFolder: Sized {
                             || j.fields.iter().any(|f| f.is_stripped());
```

Comment:
\[Self note (for future):\] We can probably remove this part because variant fields always have inherited visibility.